examples/blame_example.ipynb

```diff
@@ -67,212 +64,233 @@
+    "for _ in range(20): #Attempt 20 simulations looking for a collision\n",
     "    light_response = iai.light(location=location)\n",
     "\n",
     "    response = iai.initialize(\n",
-    "        location=location,  \n",
-    "        agent_count=25,    \n",
-    "        get_birdview=True,  \n",
-    "        traffic_light_state_history=[light_response.traffic_lights_states]  \n",
+    "        location=location,\n",
+    "        agent_count=15,\n",
+    "        get_birdview=True,\n",
+    "        traffic_light_state_history=[light_response.traffic_lights_states]\n",
     "    )\n",
-    "    agent_attributes = response.agent_attributes  \n",
-    "    scene_plotter.initialize_recording(response.agent_states, \n",
-    "                                       agent_attributes=agent_attributes, \n",
+    "    agent_attributes = response.agent_attributes\n",
+    "    scene_plotter.initialize_recording(response.agent_states,\n",
+    "                                       agent_attributes=agent_attributes,\n",
     "                                       traffic_light_states=light_response.traffic_lights_states)\n",
     "\n",
     "    agent_state_history = []\n",
     "    traffic_light_state_history = []\n",
     "\n",
-    "    # 20-second scene\n",
-    "    for _ in range(200):  \n",
+    "    # 10-second scene\n",
+    "    SIMULATION_LENGTH = 100\n",
+    "    for t in range(SIMULATION_LENGTH):\n",
     "        light_response = iai.light(location=location, recurrent_states=light_response.recurrent_states)\n",
     "        response = iai.drive(\n",
     "            location=location,\n",
     "            agent_attributes=agent_attributes,\n",
     "            agent_states=response.agent_states,\n",
     "            recurrent_states=response.recurrent_states,\n",
-    "            get_birdview=True,\n",
+    "            get_birdview=False,\n",
     "            traffic_lights_states=light_response.traffic_lights_states,\n",
     "            get_infractions=True,\n",
     "            random_seed=1\n",
     "        )\n",
     "        scene_plotter.record_step(response.agent_states, traffic_light_states=light_response.traffic_lights_states)\n",
     "        agent_state_history.append(response.agent_states)\n",
     "        traffic_light_state_history.append(light_response.traffic_lights_states)\n",
-    "    \n",
-    "    \n",
-    "    blame_response = iai.blame(\n",
-    "        location=location,\n",
-    "        candidate_agents=[0, 0],\n",
-    "        agent_state_history=agent_state_history,\n",
-    "        agent_attributes=agent_attributes,\n",
-    "        traffic_light_state_history=traffic_light_state_history,\n",
-    "        get_birdviews=True,\n",
-    "        detect_collisions=True\n",
-    "    )\n",
-    "    collisions = blame_response.blamed_collisions\n",
-    "    \n",
+    "\n",
+    "        #Check for collisions and acquire: agent IDs, collision start time, collision end time\n",
```

Comment:
What if a pair of agents have two collisions in the scenario? i.e. They collide, and separate, and collide again?

Reply:
I have added a fix to that so the same collision pair can be added multiple times.